bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerFragmentReplicator.java

```diff
@@ -473,4 +496,17 @@ private static void updateEnsembleInfo(
                 }
             });
     }
+
+    static class Throttler {
+        private final RateLimiter rateLimiter;
+
+        Throttler(int throttleBytes) {
```

Comment:
Use `permits` instead of `throttleBytes` ?

Reply:
Permits is used when acquire.